test/src/main/java/org/apache/accumulo/test/VolumeChooserIT.java

```diff
@@ -132,194 +137,177 @@ public void configure(MiniAccumuloConfigImpl cfg, Configuration hadoopCoreSite)
-    assertEquals(26, fileCount, "Wrong number of files");
+    return TableId.of(client.tableOperations().tableIdMap().get(tableName));
   }
 
-  private void configureNamespace(AccumuloClient accumuloClient, String volumeChooserClassName,
-      String configuredVolumes, String namespace) throws Exception {
-    accumuloClient.namespaceOperations().create(namespace);
-    // Set properties on the namespace
-    accumuloClient.namespaceOperations().setProperty(namespace, PERTABLE_CHOOSER_PROP,
-        volumeChooserClassName);
-    accumuloClient.namespaceOperations().setProperty(namespace, PREFERRED_CHOOSER_PROP,
-        configuredVolumes);
+  private void createNamespaceWithPreferredChooser(AccumuloClient client, String namespace,
+      Path preferredVolume) throws Exception {
+    client.namespaceOperations().create(namespace);
+    client.namespaceOperations().setProperty(namespace, PERTABLE_CHOOSER_PROP,
+        PreferredVolumeChooser.class.getName());
+    if (preferredVolume != null) {
+      client.namespaceOperations().setProperty(namespace, PREFERRED_CHOOSER_PROP,
+          preferredVolume.toString());
+    }
   }
 
-  private void verifyVolumesForWritesToNewTable(AccumuloClient accumuloClient, String myNamespace,
-      String expectedVolumes) throws Exception {
-    String tableName = myNamespace + ".1";
-
-    accumuloClient.tableOperations().create(tableName);
-    TableId tableID = TableId.of(accumuloClient.tableOperations().tableIdMap().get(tableName));
+  private void createNamespaceWithRandomChooser(AccumuloClient client, String namespace)
+      throws Exception {
+    client.namespaceOperations().create(namespace);
+    client.namespaceOperations().setProperty(namespace, PERTABLE_CHOOSER_PROP,
+        RandomVolumeChooser.class.getName());
+    // The random volume chooser should ignore this property
+    client.namespaceOperations().setProperty(namespace, PREFERRED_CHOOSER_PROP, "ignored");
+  }
 
-    // Add 10 splits to the table
-    addSplits(accumuloClient, tableName);
-    // Write some data to the table
-    writeAndReadData(accumuloClient, tableName);
-    // Verify the new files are written to the Volumes specified
-    verifyVolumes(accumuloClient, TabletsSection.getRange(tableID), expectedVolumes);
+  private void createTableAndVerifyVolumesUsed(AccumuloClient client, String namespace,
+      Path... expectedVolumes) throws Exception {
+    String tableName = namespace + ".1";
+    TableId tableID = createAndVerifyTable(client, tableName, alpha_rows, true);
+
+    // Verify the new files are written only to the expected volumes
+    Set<String> allTableFiles = getServerContext().getAmple().readTablets().forTable(tableID)
+        .fetch(ColumnType.FILES).build().stream().flatMap(tm -> tm.getFiles().stream())
+        .map(StoredTabletFile::getPath).map(Path::toString).collect(toSet());
+    assertEquals(26, allTableFiles.size(), "Wrong number of files");
```

Comment:
```suggestion
    assertEquals(alpha_rows.size(), allTableFiles.size(), "Wrong number of files");
```
I think this was here before you changed things but could use `allTableFiles.size()` instead of 26 here